src/dragonball/src/dbs_upcall/src/lib.rs

```diff
@@ -356,21 +397,24 @@ impl<S: UpcallClientService + Send> UpcallEpollHandler<S> {
                     info.set_state(UpcallClientState::ServiceConnected);
                 }
             }
-            UpcallClientState::ServiceBusy => match info.handle_response() {
-                Ok(response) => {
-                    trace!("upcall handle response success");
-                    info.set_state(UpcallClientState::ServiceConnected);
-                    info.consume_callback(response);
-                }
-                Err(e) => {
-                    warn!("upcall response failed {}", e);
-                    info.set_state(UpcallClientState::WaitingServer);
-                    if let Err(e) = self.set_reconnect() {
-                        error!("set reconnect error: {}", e);
-                        info.set_state(UpcallClientState::ReconnectError);
+            UpcallClientState::ServiceBusy => {
+                match info.handle_response() {
+                    Ok(response) => {
+                        trace!("upcall handle response success");
+                        info.set_state(UpcallClientState::ServiceConnected);
+                        info.consume_callback(response);
+                    }
+                    Err(e) => {
+                        warn!("upcall response failed {}", e);
+                        info.set_state(UpcallClientState::WaitingServer);
+                        if let Err(e) = self.set_reconnect() {
+                            error!("set reconnect error: {}", e);
+                            info.set_state(UpcallClientState::ReconnectError);
+                        }
                     }
                 }
-            },
+                self.metrics.handle_response_count.inc();
```

Comment:
I think the metric for how many response we handled in upcall is not very useful..

Reply:
Okay, I removed it.